.meta/dietpi-imager

```diff
@@ -102,8 +102,8 @@
 
 		# Detect partitions and list for selection
 		G_WHIP_MENU_ARRAY=($(lsblk -npo NAME,SIZE ${FP_SOURCE}?*))
-		# - Visually separate dev name and space
-		for ((i=1;i<${#G_WHIP_MENU_ARRAY[@]};i+=2)); do G_WHIP_MENU_ARRAY[$i]=": ${G_WHIP_MENU_ARRAY[$i]}"; done
+		# - Visually separate dev name and size and add FS type
+		for ((i=1;i<${#G_WHIP_MENU_ARRAY[@]};i+=2)); do G_WHIP_MENU_ARRAY[$i]=": $(lsblk -drno SIZE,FSTYPE ${G_WHIP_MENU_ARRAY[$i-1]})"; done
```

Comment:
@sal666 
Somhow I don't like the doubled `lsblk -o SIZE`, but also I don't have a better solution for now, besides doing a `while read` loop through a single lsblk output, but that costs much more lines :thinking:.